Apps/WebClient/src/ClientApp/app/components/timeline/comment.vue

```diff
@@ -4,28 +4,253 @@
 .comment-body {
   background-color: $lightGrey;
   border-radius: 10px;
+  display: flex;
+}
+
+.editing {
+  background-color: lightyellow;
+}
+
+.commnet-menu {
+  color: $soft_text;
+}
+
+.comment-text {
+  white-space: pre-line;
+}
+
+.comment-input {
+  flex: 1 1 auto;
+}
+
+.comment-button {
+  flex: 0 0 auto;
+  flex-direction: row;
 }
 </style>
 <template>
   <b-col>
-    <b-row class="comment-body p-3 my-2">
-      {{ comment.text }}
+    <b-row
+      class="comment-body p-2 my-1"
+      align-v="center"
+      v-if="mode === 'text'"
+    >
+      <b-col class="comment-text">{{ comment.text }}</b-col>
+      <b-col>
+        <div class="d-flex flex-row-reverse">
+          <b-navbar-nav>
```

Comment:
not sure if nav-bar is what we want here.

Reply:
Changed to dropdown